suitcase/hdf5.py

```diff
@@ -94,8 +95,27 @@ def export(headers, filename, stream_name=None, fields=None, timestamps=True, us
                                                 compression='gzip',
                                                 fletcher32=True)
                     data = [e['data'][key] for e in events]
-                    dataset = data_group.create_dataset(
-                        key, data=data, compression='gzip', fletcher32=True)
+                    data = np.array(data)
```

Comment:
We know ahead of time from `value['dtype']`.  We can likely be smarter about this.